app/views/unattended/provisioning_templates/snippet/preseed_netplan_setup.erb

```diff
@@ -23,7 +23,7 @@ oses:
 <%#
 ##### Processing bond-interfaces #####
 -%>
-<%- found = false -%>
+<%- found = false; id = 0 -%>
 <%- @host.bond_interfaces.each do | bond | -%>
```

Comment:
I'm not sure you can use this in safemode, but if you can this is a lot
```suggestion
<%- @host.bond_interfaces.each_with_index do | bond, id | -%>
```

Reply:
Sadly I think it's not: https://github.com/theforeman/safemode/pull/55

Reply:
then I would stay at the original implmentation. and we only have <%= id = 0 %> in the last commit.